src/lib/containers/table/table-top/ProgrammaticOptions.tsx

```diff
@@ -48,7 +44,11 @@ function ProgrammaticOptions({
         const res = await getTransformSqlWithFacetsRequest(
           transformSqlWithFacetsRequest,
         )
-        setGeneratedSql(res.transformedSql.replace(/"/g, '\\"'))
+        // SWC-5686: The ID column is required by the client, and this column may not have been selected!
+        // Change the SQL to "SELECT * ..."
+        const indexOfFrom = res.transformedSql.toUpperCase().indexOf('FROM SYN')
+        const selectStarTransformedSql = `SELECT * ${res.transformedSql.substring(indexOfFrom)}`
+        setGeneratedSql(selectStarTransformedSql.replace(/"/g, '\\"'))
```

Comment:
Initial select sql is defined in the portal configuration, and only modified by the backend (via user interaction with selecting facets).  The user does not have the opportunity to change the sql.   Even if they did, the backend guards against SQL injection before it hits the database.